### DISCUSSION TITLE
Anybody know where I might find a code example of a floating Live streaming Widget?

### DISCUSSION THREAD

**Discussion by thorssig** [netmix/radio-station#456] [General]
Hi,
Anybody know where I might find a code example of the floating Live streaming Widget shown in the https://demo.radiostation.pro/widgets/ page?

I've tried using the floater that is configureable thru the "WP - Radio Station Pro -> Player" but it's not showing me what I need.
![Gott dæmi um Radio Station](https://github.com/netmix/radio-station/assets/133007747/e8999b17-5783-4d22-8d91-936edc8c636c)

**Comment by tonyzeoli** [comment]
I'm sorry! We weren't checking the Discussion tab here on Git. We just missed it. This is from 2023. Did you ever find what you're looking for? We do have a pop up player off the main Radio Station PRO persistent player. If it's not what you need, please describe what you needed and maybe found so we can figure it out.

**Comment by tonyzeoli** [comment]
We'll close this discussion given its age.
